src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -318,6 +318,23 @@ private void Populate(ref LocalState state, int start)
             }
         }
 
+        // PROTOTYPE(NullableReferenceTypes): Consider setting treatUnconstrainedTypeParameterAsNullable during initial binding, and removing this method.
+        private static TypeSymbolWithAnnotations GetAdjustedType(TypeSymbolWithAnnotations initialType)
+        {
+            // If the initial type was an unconstrained type parameter, we want to treat it as nullable, overriding
```

Comment:
>// [](start = 12, length = 2)

Perhaps make this an xml doc instead. 
Also, consider renaming to "FixUnconstrainedTypeParameters", as it gives some indication of purpose, whereas "adjusted type" provides no clue.